index.html

```diff
@@ -4566,11 +4587,19 @@ <h2>
             fulfillment value containing invalid data. This would potentially
             leave the payment request in an inconsistent state since the
             developer hasn't successfully handled the change event.
+          </p>
+          <p>
             Consequently, the <a>PaymentRequest</a> moves to a "<a>closed</a>"
             state. The error is signaled to the developer through the rejection
             of the <a>[[\acceptPromise]]</a>, i.e., the promise returned by
             <a data-lt="PaymentRequest.show">show()</a>.
           </p>
+          <p data-link-for="PaymentResponse">
+            Similarly, aborting the update occurs during a <a>retry()</a> will
```

Comment:
This sentence doesn't read right... "aborting the update occurs during a retry() will cause"?